### PR TITLE
Update README.md for Arduino 1.8.8

### PR DESCRIPTION
**Pull request by leoguiders**
Updated the Readme to support the changes in Arduino 1.8.8 and ESP8266 core for Arduino 2.5.0-beta2.
SAMD core instructions are untested.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,90 +1,88 @@
...
-
-Replace the arduino-builder executable from the Arduino installation folder with the appropriate file based on your OS from this link [arduino-builder-219.zip](http://downloads.arduino.cc/PR/arduino-builder/arduino-builder-219.zip).
-
-### 4. Modify the platform.txt file
+### 3. Modify the platform.txt file
 
 If you have already used the previous example code remove the linker flag `-libalgobsec` in the platform.txt file.
 
-The arduino-builder passes the linker flags under `compiler.c.elf.extra_flags`. Most platform.txt files in the combine recipe place this previously unused variable in the start of the link recipe whereas it should be near the end along with the other pre-compiled libraries flags.
+The arduino-builder passes the linker flags under `{compiler.libraries.ldflags}`. Open you platform.txt and add this near the end of your recipe.c.combine.pattern.
```
Comment (BoschSensortec) [inline README.md]: Open **your**

File: README.md [hunk 2]
```diff
@@ -1,90 +1,88 @@
...
 
 #### Examples
 
-##### ESP8266 community forum's ESP8266 core
+##### ESP8266 community forum's ESP8266 core v2.5.0-beta2
```
Comment (BoschSensortec) [inline README.md]: Would recommend leaving the standard release instructions as all users may not use the beta.

File: README.md [hunk 3]
```diff
@@ -1,90 +1,88 @@
...
 ```
 
 Should be,
 ```
 ## Combine gc-sections, archives, and objects
-recipe.c.combine.pattern="{compiler.path}{compiler.c.elf.cmd}"  "-L{build.path}" {compiler.c.elf.flags} "-T{build.variant.path}/{build.ldscript}" "-Wl,-Map,{build.path}/{build.project_name}.map" --specs=nano.specs --specs=nosys.specs {compiler.ldflags} -o "{build.path}/{build.project_name}.elf" {object_files} -Wl,--start-group {compiler.arm.cmsis.ldflags} -lm {compiler.c.elf.extra_flags} "{build.path}/{archive_file}" -Wl,--end-group
+recipe.c.combine.pattern="{compiler.path}{compiler.c.elf.cmd}"  "-L{build.path}" {compiler.c.elf.flags} {compiler.c.elf.extra_flags} "-T{build.variant.path}/{build.ldscript}" "-Wl,-Map,{build.path}/{build.project_name}.map" --specs=nano.specs --specs=nosys.specs {compiler.ldflags} -o "{build.path}/{build.project_name}.elf" {object_files} -Wl,--start-group -lm {compiler.libraries.ldflags} "{build.path}/{archive_file}" -Wl,--end-group
```
Comment (BoschSensortec) [inline README.md]: this variable now throws an error if compiling a library where this is no pre-compiled library used.

File: README.md (hunk 1 above)
Comment (BoschSensortec) [reply README.md]: Also, this variable needs to be added in the '# These can be overridden in platform.local.txt' section as 'compiler.libraries.ldflags='

**Comment by michapr** [comment]
this is working fine for me with 1.8.8 and ESP 2.5.0 final:
https://github.com/BoschSensortec/BSEC-Arduino-library/issues/40

**Comment by BoschSensortec** [comment]
Hi @leoguiders,

Thank you for the pull-request. Your changes have been incorporated into the latest release.

**Pull request closed without merging**